autoload/lsp/ui/vim/semantic.vim

```diff
@@ -15,24 +15,41 @@ function! lsp#ui#vim#semantic#is_enabled() abort
     return g:lsp_semantic_enabled && (s:use_vim_textprops || s:use_nvim_highlight) ? v:true : v:false
```

Comment:
Not required but we should probably look into moving this to https://github.com/prabirshrestha/vim-lsp/tree/master/autoload/lsp/internal

Reply:
Done in [ea2fe4c](https://github.com/prabirshrestha/vim-lsp/pull/1275/commits/ea2fe4c2328c9fe56adbdbdf2ae4f6805fd5a1e7)